utils/markdown.js

```diff
@@ -208,7 +208,7 @@ function buildTree(lines) {
         tokens.push(line.substring(indent.length));
         line = lines[++i];
       }
-      node.text = tokens.join('↵');
+      node.children = buildTree(tokens);
```

Comment:
Shouldn't you update `MarkdownNoteNode` to remove `text` property and fix all the usages?